README-dfds-data.md

```diff
@@ -0,0 +1,23 @@
+# Grafana dashboards
```

Comment:
Should we add a link to this README in the main README? Or will that create merge conflicts when pulling upstream changes?

Reply:
Afraid of merge conflicts if I do this